src/aind_metadata_upgrader/procedures_upgrade.py

```diff
@@ -0,0 +1,569 @@
+"""Module to contain code to uprgade old procedures"""
+
+import logging
+from typing import Any, Optional, Union
+
+import semver
+from aind_data_schema.core.procedures import (
+    Craniotomy,
+    FiberImplant,
+    Headframe,
+    IntraCerebellarVentricleInjection,
+    IntraCisternalMagnaInjection,
+    IntraperitonealInjection,
+    IontophoresisInjection,
+    NanojectInjection,
+    NonViralMaterial,
+    OphysProbe,
+    OtherSubjectProcedure,
+    Perfusion,
+    Procedures,
+    RetroOrbitalInjection,
+    SpecimenProcedure,
+    Surgery,
+    TarsVirusIdentifiers,
+    ViralMaterial,
+)
+from aind_data_schema.models.devices import FiberProbe
+
+from aind_metadata_upgrader.base_upgrade import BaseModelUpgrade
+from aind_metadata_upgrader.utils import (
+    check_field,
+    construct_new_model,
+    drop_unused_fields,
+    get_or_default,
+)
+
+
+class InjectionMaterialsUpgrade(BaseModelUpgrade):
+    """Handle upgrades for InjectionMaterials models."""
+
+    def __init__(self, allow_validation_errors=False):
+        """Handle upgrades for InjectionMaterials models"""
+
+        self.allow_validation_errors = allow_validation_errors
```

Comment:
you aren't calling the parent class constructor, which means it's not really being treating like a subclass. Do you really need to inherit here? Or is this just a wrapper around the `upgrade_viral_material` function?

Reply:
I think you're right that inheriting doesn't matter here. Yes, it is a wrapper, though.